packages/vitepress-plugin-git-changelog/src/vite/helpers.ts

```diff
@@ -336,7 +336,7 @@ export async function parseCommits(
     // authors
     const authors = await parseCommitAuthors(rawCommit, mapContributors)
     authors.forEach(a => allAuthors.set(a.name, omit(a, ['email'])))
-    resolvedCommit.authors = authors.map(a => a.name)
+    resolvedCommit.authors = authors
```

Comment:
I'd prefer to just pass in the name here instead of the whole AuthorInfo, as it might be more space efficient.

In changelog.vue it is possible to match avatars from changelog.authors separately.

(personal viewpoint)

Reply:
ref: https://github.com/lulu0119/integrations/pull/1

Just as an example of what I think, you are free to receive it or not.